src/hooks/useScroll.js

```diff
@@ -0,0 +1,18 @@
+import { useInView } from 'react-intersection-observer'
+import { useAnimation } from 'framer-motion'
+
+const useScroll = () => {
+  const controls = useAnimation() // animation
+  const [element, view] = useInView({ threshold: 0.3 })
+
+  // if element is in view, we are starting the show animation
```

Comment:
remove commented code

Reply:
that is a comment describing what the code is doing.